dali/test/python/jax_plugin/test_integration.py

```diff
@@ -41,7 +41,7 @@ def test_dali_tensor_gpu_to_jax_array(dtype, shape, value):
     assert jax.numpy.array_equal(jax_array, jax.numpy.full(shape, value, dtype))
 
     # Make sure JAX array is backed by the GPU
-    assert jax_array.device() == jax.devices()[0]
+    assert jax_array.device == jax.devices()[0]
```

Comment:
A breaking change?

Reply:
Slipped in... That's a breaking change... in JAX 0.4.31 :\